models/staging/stg_marketo.yml

```diff
@@ -37,9 +38,10 @@ models:
 
   - name: stg_marketo__activity_click_email
     columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: activity_id
         tests:
-          - unique
```

Comment:
Following up from our chat today, I see the staging uniqueness tests were never based on the surrogate keys, even though many staging models included them. I say we leave the tests removed for now rather than adding a uniqueness test to the new column, since we’ll need to revisit the testing approach later anyway and don’t want to introduce new issues.